tasks/FindMyDisk.tex

```diff
@@ -0,0 +1,88 @@
+\section{Find My Disk [Housekeeper]}
+The robot helps a blind person to locate a LP, compact disk, or audio-cassette (hereinafter a \textit{disk}) in a shelf. The operator will ask the robot to describe what it sees in either the shelf or the media cover, but won't allow the robot to touch her treasures.
```

Comment:
How will the disks be on the shelf? face out or spine out?

Or a mix of both and the robot needs to ask the `blind person` to flip to the next `disk`?


1. https://www.amazon.com/Hudson-Hi-Fi-Mount-Record-Display/dp/B071WKY96X
2. https://www.amazon.com/Way-Basics-2-Shelf-Storage-Espresso/dp/B075M53XKB
3. https://www.amazon.com/Vinyl-Record-Storage-Rack-Eco-Friendly/dp/B074JG9X1M/